### DISCUSSION TITLE
What all do I need from the Lazyvim repo?

### DISCUSSION THREAD

**Discussion by JadoJodo** [LazyVim/LazyVim#4482] [General]
Hi, all –

New to LazyVim/NeoVim and a very big noob to Vim in general. I have LazyVim installed and working – love it so far – but I don't know what other things I need / don't need from the LazyVim repo. Are the plugins/scripts/etc that are included in this repo also built-in to my Starter LazyVim? They aren't in my folders, nor do I see any obvious signs of them being pulled down in the init script, but I also know the starter pack shows >30 plugins by default. Are these those? Should I manually install any from this repo if I want them?

Thanks!

**Comment by dpetka2001** [comment]
Yes, everything included here is also included in the starter template. You don't need to install any additional plugins from what is mentioned in the [core plugins](https://www.lazyvim.org/plugins), except if you  want to change some default options that are not to your preferences. 

Everything that is mentioned in the [Extras](https://www.lazyvim.org/extras) is not installed by default and you would have to enable them via the `:LazyExtras` command, which will pop up an interface for you to enable/disable the ones you want.
